Directory.Build.props

```diff
@@ -73,5 +70,4 @@
 
   <Import Project="eng\Dependencies.props" />
   <Import Project="eng\ProjectReferences.props" />
-  <Import Project="eng\targets\Csharp.Refs.props" Condition="'$(IsReferenceAssemblyProject)' == 'true'" />
```

Comment:
Hey @JunTaoLuo why is eng/targets/Csharp.Refs.props still in this repo?

Reply:
No reason, i just forgot to remove it.